docker/public/Dockerfile-h2o-release

```diff
@@ -3,4 +3,4 @@ FROM openjdk:11
 RUN mkdir -p /opt/h2oai/h2o-3/
 COPY h2o.jar /opt/h2oai/h2o-3/
 
-CMD java -XX:+UseContainerSupport -jar /opt/h2oai/h2o-3/h2o.jar
+CMD java -XX:+UseContainerSupport -XX:MaxRAMPercentage=90 -jar /opt/h2oai/h2o-3/h2o.jar
```

Comment:
With this setting we should disable XGBoost, @honzasterba what do you think?

Reply:
yes, 10% is not enough